### ISSUE TITLE
There are many broken elements in this repo, after this merge all will be set !!

### ISSUE THREAD

**Issue by harshit-soora** [kumaraditya303/Library-Management-System#68]
**These are the functions that I have added:**
1. A admin toggle on register page for admin , (alternative - this feature could be in-build inside the code with some super user too whose credential are already stored when the system runs.)
2. Students list feature in admin which allow the admin to get details of students who have registered

**These are the issues that I have resolve:**
1. Admin login doesn't redirect to admin dashboard instead it redirect to normal dashboard
2. Add Book and Remove book have some query error in it which have been resolved.
3. After issue book the list show author and book name as empty (added a join query to resolve that)
4. Remove book have similar error where the drop down list doesn't have Book and Author name

There were some minor errors/bug in between, I will be adding my work under this issue. You can review and merge it.

One more courtesy, I will be adding the newer version at my Github [repo](https://github.com/harshit-soora/) for sure I will promote your work :) 

Your coding style was very nice :D

**Comment by kumaraditya303** [comment]
Hi @harshit-soora,

> A admin toggle on register page for admin , (alternative - this feature could be in-build inside the code with some super user too whose credential are already stored when the system runs.)

This should be implemented at the backend itself for security issues, this could be implemented with a cli which directly write to the db and creates super user.

> Students list feature in admin which allow the admin to get details of students who have registered

Nice feature!

> Your coding style was very nice :D 
Thanks

**Comment by harshit-soora** [comment]
Yes @kumaraditya303 for future releases you can just hard code one entry of admin directly inside the ```view.py``` file.

For my use-case, I had to explicitly show how a admin is registered that's why I kept it that why

In future that toggle could be for anything like "more privileged user like research scholar or something"

**Comment by Josh2077** [comment]
$ git clone https://github.com/kumaraditya303/Library-Managament-System.git
Cloning into 'Library-Managament-System'...
remote: Repository not found.
fatal: repository 'https://github.com/kumaraditya303/Library-Managament-System.g
it/' not found
